From a9f253299c93de63c3ec5eb211447b45dbb55dc9 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
Date: Tue, 21 May 2019 22:33:54 +0900
Subject: [PATCH 1/3] shared/sec: rename func for getting mng data

This update is to rename function for getting global management data
`spp_get_mng_data_addr()` to `sppwk_get_mng_data()`.

Signed-off-by: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
---
 .../secondary/spp_worker_th/command_proc.c    | 14 +++---
 src/shared/secondary/spp_worker_th/spp_proc.c | 45 ++++++++++---------
 src/shared/secondary/spp_worker_th/spp_proc.h | 16 +++----
 3 files changed, 38 insertions(+), 37 deletions(-)

diff --git a/src/shared/secondary/spp_worker_th/command_proc.c b/src/shared/secondary/spp_worker_th/command_proc.c
index 151aaa6..94f4142 100644
--- a/src/shared/secondary/spp_worker_th/command_proc.c
+++ b/src/shared/secondary/spp_worker_th/command_proc.c
@@ -104,7 +104,7 @@ spp_get_client_id(void)
 {
 	struct startup_param *startup_param;
 
-	spp_get_mng_data_addr(&startup_param,
+	sppwk_get_mng_data(&startup_param,
 			NULL, NULL, NULL, NULL, NULL, NULL);
 	return startup_param->client_id;
 }
@@ -115,7 +115,7 @@ spp_get_process_type(void)
 {
 	struct startup_param *startup_param;
 
-	spp_get_mng_data_addr(&startup_param,
+	sppwk_get_mng_data(&startup_param,
 			NULL, NULL, NULL, NULL, NULL, NULL);
 	return startup_param->secondary_type;
 }
@@ -235,7 +235,7 @@ spp_update_component(
 	int *change_core = NULL;
 	int *change_component = NULL;
 
-	spp_get_mng_data_addr(NULL, NULL, &comp_info_base, &core_info,
+	sppwk_get_mng_data(NULL, NULL, &comp_info_base, &core_info,
 				&change_core, &change_component, NULL);
 
 	switch (wk_action) {
@@ -387,7 +387,7 @@ spp_update_port(enum sppwk_action wk_action,
 				"(component = %s)\n", name);
 		return SPP_RET_NG;
 	}
-	spp_get_mng_data_addr(NULL, NULL,
+	sppwk_get_mng_data(NULL, NULL,
 			&comp_info_base, NULL, NULL, &change_component, NULL);
 	comp_info = (comp_info_base + component_id);
 	port_info = get_sppwk_port(port->iface_type, port->iface_no);
@@ -491,7 +491,7 @@ spp_flush(void)
 	int ret = SPP_RET_NG;
 	struct cancel_backup_info *backup_info = NULL;
 
-	spp_get_mng_data_addr(NULL, NULL, NULL,
+	sppwk_get_mng_data(NULL, NULL, NULL,
 				NULL, NULL, NULL, &backup_info);
 
 	/* Initial setting of each interface. */
@@ -540,7 +540,7 @@ spp_iterate_core_info(struct spp_iterate_core_params *params)
 		}
 
 		for (cnt = 0; cnt < core->num; cnt++) {
-			spp_get_mng_data_addr(NULL, NULL, &comp_info_base,
+			sppwk_get_mng_data(NULL, NULL, &comp_info_base,
 							NULL, NULL, NULL, NULL);
 			comp_info = (comp_info_base + core->id[cnt]);
 #ifdef SPP_VF_MODULE
@@ -602,7 +602,7 @@ sppwk_get_ethdev_port_id(enum port_type iface_type, int iface_no)
 {
 	struct iface_info *iface_info = NULL;
 
-	spp_get_mng_data_addr(NULL, &iface_info,
+	sppwk_get_mng_data(NULL, &iface_info,
 				NULL, NULL, NULL, NULL, NULL);
 	switch (iface_type) {
 	case PHY:
diff --git a/src/shared/secondary/spp_worker_th/spp_proc.c b/src/shared/secondary/spp_worker_th/spp_proc.c
index 4105fb8..57b6321 100644
--- a/src/shared/secondary/spp_worker_th/spp_proc.c
+++ b/src/shared/secondary/spp_worker_th/spp_proc.c
@@ -1020,29 +1020,30 @@ int spp_set_mng_data_addr(struct startup_param *startup_param_addr,
 	return SPP_RET_OK;
 }
 
-/* Get manage data address */
-void spp_get_mng_data_addr(struct startup_param **startup_param_addr,
-			   struct iface_info **iface_addr,
-			   struct spp_component_info **component_addr,
-			   struct core_mng_info **core_mng_addr,
-			   int **change_core_addr,
-			   int **change_component_addr,
-			   struct cancel_backup_info **backup_info_addr)
+/* Get manage data from global var for given non-NULL args. */
+void sppwk_get_mng_data(
+		struct startup_param **startup_param_p,
+		struct iface_info **iface_p,
+		struct spp_component_info **component_p,
+		struct core_mng_info **core_mng_p,
+		int **change_core_p,
+		int **change_component_p,
+		struct cancel_backup_info **backup_info_p)
 {
 
-	if (startup_param_addr != NULL)
-		*startup_param_addr = g_mng_data_addr.p_startup_param;
-	if (iface_addr != NULL)
-		*iface_addr = g_mng_data_addr.p_iface_info;
-	if (component_addr != NULL)
-		*component_addr = g_mng_data_addr.p_component_info;
-	if (core_mng_addr != NULL)
-		*core_mng_addr = g_mng_data_addr.p_core_info;
-	if (change_core_addr != NULL)
-		*change_core_addr = g_mng_data_addr.p_change_core;
-	if (change_component_addr != NULL)
-		*change_component_addr = g_mng_data_addr.p_change_component;
-	if (backup_info_addr != NULL)
-		*backup_info_addr = g_mng_data_addr.p_backup_info;
+	if (startup_param_p != NULL)
+		*startup_param_p = g_mng_data_addr.p_startup_param;
+	if (iface_p != NULL)
+		*iface_p = g_mng_data_addr.p_iface_info;
+	if (component_p != NULL)
+		*component_p = g_mng_data_addr.p_component_info;
+	if (core_mng_p != NULL)
+		*core_mng_p = g_mng_data_addr.p_core_info;
+	if (change_core_p != NULL)
+		*change_core_p = g_mng_data_addr.p_change_core;
+	if (change_component_p != NULL)
+		*change_component_p = g_mng_data_addr.p_change_component;
+	if (backup_info_p != NULL)
+		*backup_info_p = g_mng_data_addr.p_backup_info;
 
 }
diff --git a/src/shared/secondary/spp_worker_th/spp_proc.h b/src/shared/secondary/spp_worker_th/spp_proc.h
index dffa6a5..921aead 100644
--- a/src/shared/secondary/spp_worker_th/spp_proc.h
+++ b/src/shared/secondary/spp_worker_th/spp_proc.h
@@ -689,7 +689,7 @@ int spp_set_mng_data_addr(struct startup_param *startup_param_addr,
 /**
  * Get mange data address
  *
- * @param iface_addr
+ * @param startup_param_p
  *  g_startup_param write address
  * @param iface_addr
  *  g_iface_info write address
@@ -704,12 +704,12 @@ int spp_set_mng_data_addr(struct startup_param *startup_param_addr,
  * @param backup_info_addr
  *  g_backup_info write address
  */
-void spp_get_mng_data_addr(struct startup_param **startup_param_addr,
-			   struct iface_info **iface_addr,
-			   struct spp_component_info **component_addr,
-			   struct core_mng_info **core_mng_addr,
-			   int **change_core_addr,
-			   int **change_component_addr,
-			   struct cancel_backup_info **backup_info_addr);
+void sppwk_get_mng_data(struct startup_param **startup_param_p,
+		struct iface_info **iface_p,
+		struct spp_component_info **component_p,
+		struct core_mng_info **core_mng_p,
+		int **change_core_p,
+		int **change_component_p,
+		struct cancel_backup_info **backup_info_p);
 
 #endif /* _SPP_PROC_H_ */
-- 
2.17.1

